### PR TITLE
#171434652 Fix user interface styles

### PR DESCRIPTION
**Pull request by victkarangwa**
#### What does this PR do?
- Fix the delete comment pop up style
- Fix broken user table styles
#### Description of Task to be completed?
- This PR fixes the styles for the table component and the popup styles for deleting comments
#### How should this be manually tested?
- `git clone https://github.com/andela/team-odd-bn-frontend.git`
- `cd team-odd-bn-backend`
- git checkout bg-fix-user-interface-171434652
- `npm install`
- `npm run dev` to run server
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#171434652](https://www.pivotaltracker.com/story/show/171434652)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

**Review by NiyongaboEric — APPROVED** [review]
Good job @victorkarangwa4

**Review by nziokaivy — COMMENTED** [review]
Great job @victorkarangwa4

File: src/assets/css/createProfile.scss [hunk 1]
```diff
@@ -241,5 +245,5 @@ hr {
         width: 400px;
         height: 70px;
     }
-    
+    } 
 }
```
Comment (hezronkimutai) [inline src/assets/css/createProfile.scss:249]: eol

File: src/assets/css/profile.scss [hunk 2]
```diff
@@ -239,4 +237,4 @@ hr {
         height: 70px;
     }
     
-}
+}
```
Comment (hezronkimutai) [inline src/assets/css/profile.scss:240]: eol

**Review by hezronkimutai — APPROVED** [review]

File: src/containers/admin/UserRoles.js [hunk 3]
```diff
@@ -50,6 +50,8 @@ export class UserRoles extends Component {
       && adminState.allUsers.users.data;
 
     searchData = adminState.search.payload;
+    console.log(searchData);
+
```
Comment (kipropbrian) [inline src/containers/admin/UserRoles.js]: Is this necessary

**Review by william000000 — COMMENTED** [review]
Just remove that console @kipropbrian mentioned otherwise, it looks good

**Review by william000000 — APPROVED** [review]

**Pull request merged**